addons/sales_team/views/crm_tag_views.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<odoo>
+    <!--
+        crm.tag views
+    -->
+    <record id="sales_team_tag_form" model="ir.ui.view">
+        <field name="name">sales.team.tag.form</field>
+        <field name="model">crm.tag</field>
+        <field name="arch" type="xml">
+            <form string="Tags">
+                <sheet>
+                    <div class="oe_title">
+                        <div class="oe_edit_only">
+                            <label for="name"/>
+                        </div>
+                        <h1>
+                            <field name="name"/>
+                        </h1>
+                    </div>
+                    <group>
+                        <group>
+                            <field name="color" required="True"/>
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="sales_team_tag_tree" model="ir.ui.view">
+        <field name="name">sales.team.tag.tree</field>
```

Comment:
I know you moved this, but while we are at it, better to apply the guidelines : 
id="module_model_view_viewtype" -> id="sales_team_crm_tag_view_tree"
name="like_id_but_with_dot_instead" -> name="sales.team.crm.tag.view.tree"